src/pages/signin/Socials.js

```diff
@@ -35,27 +34,30 @@ const socialsList = [
 ];
 
 const Socials = () => (
-    <Text>
+    <View style={[styles.flexRow, styles.flexWrap]}>
         {_.map(socialsList, (social) => (
-            <Hoverable key={social.link}>
-                {(hovered) => (
-                    <View>
-                        <TextLink
-                            style={styles.pr1}
-                            href={social.link}
-                        >
-                            <Icon
-                                src={social.iconURL}
-                                height={variables.iconSizeLarge}
-                                width={variables.iconSizeLarge}
-                                fill={hovered ? themeColors.link : themeColors.textLight}
-                            />
-                        </TextLink>
-                    </View>
+            <PressableWithoutFeedback
+                key={social.link}
+                href={social.link}
+                onPress={(e) => {
+                    e.preventDefault();
+                    Link.openExternalLink(social.link);
+                }}
+                accessible={false}
+                style={[styles.mr1, styles.mt1]}
+                shouldUseAutoHitSlop={false}
```

Comment:
mr1 is for spacing between icons. mt1 is the row spacing in cases icons are wrapped
When `shoudUseAuthHitSlop` is enabled, touch area gets larger than its size. To prevent this, we need to disable it

Reply:
mr1 is for spacing between icons. mt1 is for row spacing in cases icons are wrapped.
If `shouldUseAutoHitSlop` is enabled, touch area of the pressable gets larger. To prevent this, we need to disable it